### PR TITLE
draft for initial publication, based on mission statement

### PR DESCRIPTION
**Pull request by floe**

### CODE REVIEW

**Comment by floe** [comment]
Needs the bib file, won't even try to compile otherwise...

File: publication.qmd [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+title: "The Journal of Visualization and Interaction"
+bibliography: biblio.bib
+subtitle: A [diamond open-access](https://en.wikipedia.org/wiki/Open_access#Diamond/platinum_OA) journal for the [InfoVis](http://ieeevis.org/) and [HCI](https://chi.acm.org/) communities.
+author: 
+  - "[Lonni Besançon](https://lonnibesancon.me/), Linköping University"
+  - "[Florian Echtler](https://floe.butterbrot.org/), Aalborg University"
+  - "[Matthew Kay](https://www.mjskay.com/), Northwestern University"
+  - "Chat Wacharamanotham, Swansea University"
```
Comment (chatchavan) [inline publication.qmd:9]: Could you add the link to my homepage: https://chatchavan.github.io

File: publication.qmd [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+format:
+  pdf:
+    link-citations: true
+---
+
+# What is JoVI?
```
Comment (chatchavan) [inline publication.qmd:15]: Is there a way to provide section number in the generated PDF? The numbered sections are easier to refer to during the reviewing process.

File: publication.qmd (hunk 1 above)
Comment (floe) [reply publication.qmd]: done

File: publication.qmd (hunk 2 above)
Comment (floe) [reply publication.qmd:16]: done

**Review by chatchavan — APPROVED** [review]

**Comment by mjskay** [comment]
Should this be on the webpage repo or an article in a separate repo?

**Comment by floe** [comment]
Hm. Good question, actually. I was thinking it makes the most sense in here because it's 90% the same content.

**Comment by mjskay** [comment]
I guess I was thinking this would be good as a testbed of the publication process, so was assuming we would run it through in the same way we would do a paper submission.

**Comment by floe** [comment]
OK, but for the "issue 0" publication, we just need a PDF we can shunt through OJS, right? (Otherwise we can't start the DOI process from the Royal Danish Library.) And having a separate repo is something that's only relevant for the experimental track, which I assumed we can only start once we have the rest of the "regular" process lined up...

**Comment by floe** [comment]
JFYI, I uploaded the current state of the publication.qmd file as a PDF on our OJS system and moved it to the copyediting stage. We can still make changes after our board meeting tomorrow, but most things should be prepared for now, so we can quickly go to the actual publication stage afterwards.

**Comment by floe** [comment]
Since we didn't get any further comments, I would now move the mission statement through the OJS process. @mjskay, @lonnibesancon  - could you review this PR, so that we can merge the publication version of the statement into the main repo?

**Comment by floe** [comment]
P.S. alternative idea: is it possible to merge the changes in `publication.qmd` into `index.qmd` and have them only render in the PDF version? That might be even smoother.

**Comment by floe** [comment]
I'm closing this one, please review https://github.com/journalovi/journalovi.github.io/pull/9 instead.

**Pull request closed without merging**